### ISSUE TITLE
When updating a record, It seems there is no data being posted to server.

### ISSUE THREAD

**Issue by ugoa** [ember-admin/ember-cli-admin#440]
So I am using:
`ember-cli`: `2.0.0`.
`ember-cli-admin`: `0.6.5` latest version.
`Rails`: `4.2.3` as backend API provider.
`JSONAPIAdapter` 
When I tried to edit a `account` then post it to localhost server, the backend log shows the `params` variable:
```
(#<Api::V1::User::AccountsController>)> params
=> {"controller"=>"api/v1/user/accounts", "action"=>"update", "id"=>"2", "account"=>{}}
```
You can see there is nothing in the `"account" => {}`, so I wander why this would happen. Did I miss anything that could cause the `ember-cli-admin` posting back a empty dataset?

Here are my `app/model/account.js`:
```
 import DS from 'ember-data';

 export default DS.Model.extend({
   name: DS.attr('string'),
   email: DS.attr('string')
});
```
and my `app/adapters/application.js`:
```
import DS from 'ember-data';

export default DS.JSONAPIAdapter.extend({
  namespace: 'api/v1',
  host: 'http://localhost:3000'
});
```
Correct: When checking the `Request Payload` throught the browser console, I found the data is actually there in the standard JSONAPI format:

```
data: {id: "2", attributes: {name: "Updated name", email: "UpdatedEmail@ms.com"}, type: "accounts"}
```
Now it doesn't seem like an issue in `ember-cli-admin` but more like a problem about how the rails deserialize the jsonapi format request. Anyway, if someone has done the similar stuff, I would really appreciate if you can share your experience. Thanks very much!

**Comment by OpakAlex** [comment]
hi.. we never use JSONAPI in admin. but we will be :)

**Comment by ugoa** [comment]
@OpakAlex I changed the admin adapter to JSONAPI adapter and it works.

So I've fixed this issue. Turns out that if you send a AJAX POST request. Rails will wrap the data in `request.raw_post` rather than `params`, so in order to get data, I need to do
```
information = request.raw_post
data_parsed = JSON.parse(information)
```
Anyway, there some SO answers explain why it behaves like this:
http://stackoverflow.com/questions/12221634/get-a-json-as-post-parameter
http://stackoverflow.com/questions/10494574/what-is-the-difference-between-form-data-and-request-payload
http://stackoverflow.com/questions/23118249/whats-the-difference-between-request-payload-vs-form-data-as-seen-in-chrome
